aws/mage-prod.tfvars

```diff
@@ -7,3 +7,9 @@ common_tags = {
   Service   = "mage"
   Repo      = "nursa-com/mage-ai-terraform-templates"
 }
+whitelist_cidr_blocks = [
```

Comment:
Can we put these in a secret or hide them somehow? I don't like the idea of our public ips being stored in the repo

Reply:
Let me look into putting them in secrets

Reply:
One thing I worry about is when the IPs change we'll need to know what one to remove so we'll need them labeled somehow. I could store them in secrets as a key/value pair then the terraform would have to reference each ip in the terraform.

Reply:
That could work